rules-topics/15_smithing.md

```diff
@@ -0,0 +1,99 @@
+
+### Tier VIII
+
+#### Expandable Weapon
+> A marvel of intricate crafting, this weapon modification is designed to allow the wielder the ultimate flexibility in controlling their reach. With 60 seconds of roleplay, the wielder may change the length of this weapon, from up to a Two-Handed weapon down to a Short length one-handed weapon. This allows the wielder to use this item with a Short, Long, or Two-Handed weapon rep interchangably.
+#### Recycling Kit
+> This kit is designed to assist in helping a Recycler be more efficient with their reclamation of materials. When a kit is made, a single Crafting Type is chosen from Alchemy, Smithing or Rune Carving. When the user recycles an item from that type, they may expend a charge from this kit to recover 1/2 of the ranks in materials used to construct it, rounded down to a minimum of one, instead of the usual 1/3. In the case of Recycling an Artificing item, the user only benefits from this increase for the materials of the chosen type, though multiple kits may be used in tandem. This item has 3 charges.
+#### Trickleblade Modification
+> This modification can be applied to any weapon, adding a resevoir and specially treated channels to the weapon that allow Toxins to flow along the edge of the blade. The wielder of this blade may apply Ingestible Toxins to this as if they were Weapon Coatings.
+
+
+
+### Tier IX
+
+#### Armor Repair Kit
+   >This small kit has everything an individual needs to help fix all dents, scratches, tears, breaks, and other mishaps! This kit will let the user, on a three count, refit an armor to its full value upon use. It has no effect on destroyed armor. This functions as per Rapid Refit. This item is consumed on use.
+#### Forgemaster's Hammer
+> An item that indicates the creator is nearing true Mastery, this marvel of smithing ingenuity is the key tool in the ongoing battle against wear and tear. The user may restore a Destroyed Item within 5 minutes of its destruction by declaring "Reforge". This process is not without consequence, and the item loses 2 Durability. An Item with 2 or less Durability cannot benefit from this effect. This item is consumed on use.
+#### Risky Smithing
+   >Daring to push the boundaries of their craft, this technique is reserved for those with more skill than sense. By pushing their skill to the limit, this technique can grant a weapon, shield or armor of any quality an Enchantment slot. This will give a slot to a non-Masterwork item, or increase the maximum for a Masterwork item by one. However, this does not come without cost, and cut's the item's maximum durability in half, rounded down. This can be applied to the same item multiple times, but if it would reduce the durability to 0, or below the item's remaining durability, the item is destroyed. This cannot be applied to Destroyed items.
```

Comment:
`cut's` looks like a possible typo?

Can we clarify a maximum number of enchantment slots allowed on any item?

Reply:
In theory there is no maximum, but it would be decided by the durability. If you keep halving rounding down, you rapidly hit 0.